aptos-move/aptos-gas/src/transaction/mod.rs

```diff
@@ -79,28 +79,32 @@ crate::params::define_gas_parameters!(
             GAS_SCALING_FACTOR
         ],
         // Gas Parameters for reading data from storage.
-        [load_data_base: InternalGas, "load_data.base", 16_000],
         [
-            load_data_per_byte: InternalGasPerByte,
-            "load_data.per_byte",
-            1_000
+            storage_io_per_state_slot_read: InternalGasPerArg,
```

Comment:
@vgao1996 I assume it's okay to change the unit from `InternalGas` to `InternalGasPerArg`?

Reply:
Totally okay. It's also known to me that there is currently some friction with converting back and forth between `SomeUnit` and `SomeUnitPerSomeOtherUnit` and I'll see what I can do.